src/backgroundScript.js

```diff
@@ -1,6 +1,8 @@
 const backgroundScript = (selectors, colors) => {
   let count = 0;
 
+  if (!selectors.length) return 0;
```

Comment:
Move it above count declaration?

Reply:
or just return count, which one do you prefer?

Reply:
main idea is not to allocate memory if we wont use it

Reply:
done